Header header
geometry_msgs/Pose pose
bool gripper_state
int8 stage